marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/mapping-functions/parseDate.sjs

```diff
@@ -22,7 +22,10 @@ function testMoreParseDate() {
     test.assertEqual(xs.date("1996-07-16"), core.parseDate("1996.07.16", "yyyy.MM.dd")),
 
     test.assertEqual(null, core.parseDate("07.16.1996", "Mon DD,YYYY")),
-    test.assertEqual(null, core.parseDate("notADate", "Mon DD,YYYY"))
+    test.assertEqual(null, core.parseDate("notADate", "Mon DD,YYYY")),
+    test.assertEqual(null, core.parseDate(null, "Mon DD,YYYY")),
```

Comment:
Throw in a test case for "undefined" as well - shouldn't require any code changes.